requirements.txt

```diff
@@ -5,6 +5,7 @@ anyio==3.7.1
 APScheduler==3.9.1.post1
 asyncpg==0.27.0
 boto3==1.26.1
+certifi==2024.8.30
```

Comment:
@Vagoasdf is there a reason we're not using the latest?

Reply:
This one was compatible with Requests `2.32.3` which i believe is the one we are currently running (throught FastAPI)

Reply:
Makes sense, thanks for the explanation!